mods/sbz_progression/init.lua

```diff
@@ -4,28 +4,23 @@ local modpath = minetest.get_modpath("sbz_progression")
 dofile(modpath .. "/questbook.lua")
 dofile(modpath .. "/annoy.lua")
 
-function displayDialougeLine(player_name, text)
+function displayDialogueLine(player_name, text)
     minetest.chat_send_player(player_name, "⌠ " .. text .. " ⌡")
     minetest.sound_play("dialouge", {
         to_player = player_name,
         gain = 1.0,
     })
 end
 
-function displayGlobalDialougeLine(text)
+function displayGlobalDialogueLine(text)
     minetest.chat_send_all("⌠ " .. text .. " ⌡")
     minetest.sound_play("dialouge", {
         gain = 1.0,
     })
 end
 
-minetest.register_chatcommand("cheat_hacker", {
```

Comment:
wait there used to be an achievment asociated with this command, is it still in the game

Reply:
no, i dont think so